pkg/databind/internal/secrets/cyberarkapi_test.go

```diff
@@ -0,0 +1,100 @@
+package secrets
+
+import (
+	"fmt"
+	"github.com/newrelic/infrastructure-agent/pkg/databind/pkg/data"
+	. "net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+func TestCyberArkAPI(t *testing.T) {
+
+	ts := httptest.NewServer(HandlerFunc(func(w ResponseWriter, r *Request) {
+		fmt.Fprintln(w, `{
+  "Content": "CI<s0H9>k2zUPXM",
+  "PolicyID": "ALL-APPX-WIN-SL-XNCVRRI-DEFAULT",
+  "SequenceID": "2",
+  "UserName": "testuser",
+  "CPMStatus": "success",
+  "Folder": "Root",
+  "Safe": "ALL-NERE-WIN-A-NEWRELIC-UP",
+  "Address": "localhost",
+  "Name": "ALL-localhost-testuser",
+  "LogonDomain": "localhost",
+  "DeviceType": "Operating System",
+  "LastTask": "ChangeTask",
+  "RetriesCount": "-1",
+  "LastSuccessChange": "1589288852",
+  "CreationMethod": "PVWA",
+  "PasswordChangeInProcess": "False"
+}`)
+	}))
+
+	defer ts.Close()
+	apiStruct := CyberArkAPI{
+		HTTP: &(http{
+			URL:     ts.URL,
+			Headers: make(map[string]string),
+		}),
+	}
+
+	g := CyberArkAPIGatherer(&apiStruct)
+
+	r, err := g()
+	if err != nil {
+		t.Errorf("api call failed: %v ", err)
+	}
+	fmt.Fprintf(os.Stderr, "TestCyberArkAPI: got %v\n\n", r)
```

Comment:
The same here, why do we need to do Stderr output in tests for success flow?

Reply:
Same.

Reply:
Fixed